src/Datadog.Trace.ServiceFabric/Propagators/B3ServiceFabricContextPropagator.cs

```diff
@@ -1,73 +0,0 @@
-using System;
-using System.Collections.Generic;
-using System.Text;
-using Datadog.Trace.Propagation;
-using Microsoft.ServiceFabric.Services.Remoting.V2;
-
-namespace Datadog.Trace.ServiceFabric.Propagators
-{
-    internal class B3ServiceFabricContextPropagator : IServiceFabricContextPropagator
```

Comment:
The context propagators for Service Fabric are lost during this merge. Do you want to add it back in this PR or in a separate PR?

Reply:
I believe that DD have refactored to use https://github.com/pellared/opentelemetry-dotnet-instrumentation/blob/upstream-2021-03-31/src/Datadog.Trace/Propagation/B3SpanContextPropagator.cs instead

@RassK Can you double-check if I am not mistaken, please?

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/99/commits/0ad53ac32195901c30a356bb3e6c80b39da9267e according to this merge, everything seems correct. The original code is already based on existing propagator, so special SF propagators must be removed.